### ISSUE TITLE
FATA[0000] initializing source ... connect: manifest unknown

### ISSUE THREAD

**Issue by oscni** [containers/skopeo#2455]
Don't know if this is a problem with the registry or a problem with skopeo. Using skopeo alpine package version 1.15.1

This works as expected
skopeo copy docker://docker.redpanda.com/redpandadata/console:v2.7.2 docker://<PRIVATE_REGISTRY_PATH>

This gives an error
skopeo copy docker://docker.redpanda.com/redpandadata/connect:4.39 docker://<PRIVATE_REGISTRY_PATH>
FATA[0000] initializing source docker://docker.redpanda.com/redpandadata/connect:4.39: reading manifest 4.39 in docker.redpanda.com/redpandadata/connect: manifest unknown

work-around is to use docker pull, docker tag, docker push but it seems mysterious to why it works with docker cli command and not skopeo. We're using skopeo for lots of other jobs and it has never failed, this is the first time I've seen something like this. Anyone knows what this might be or how to troubleshoot it further?

Thanks for a great project! :smiley: :tada:

**Comment by mtrmac** [comment]
Thanks for reaching out.

I see
```console
% skopeo … copy --all docker://docker.redpanda.com/redpandadata/connect:4.39 dir:t
Getting image list signatures
Copying 4 images generated from 4 images in list
Copying image sha256:678d71a1c67d45b2306ed35fec4ac1e8964873b880744f7c5ab232fdb8bfda89 (1/4)
FATA[0001] copying image 1/4 from manifest list: determining manifest MIME type for docker://docker.redpanda.com/redpandadata/connect:4.39: reading manifest sha256:678d71a1c67d45b2306ed35fec4ac1e8964873b880744f7c5ab232fdb8bfda89 in docker.redpanda.com/redpandadata/connect: received unexpected HTTP status: 500 Internal Server Error 
```

and which entry fails _varies_ from run to run (although I have always seen a 500 status).

An “internal server error” must, pretty much by definition, be diagnosed server-side.

**Comment by oscni** [comment]
yeah 500 is server side error but it of course depends on the input the server receives. How can docker pull work but not skopeo?

**Comment by mtrmac** [comment]
I don’t know. The server knows. This can be investigated server-side; without that, we are restricted to guessing things that might make a difference.

**Comment by mtrmac** [comment]
Please reopen of more information pointing at Skopeo becomes available.
